cognite/client/_api/functions.py

```diff
@@ -214,6 +214,7 @@ def create(
         index_url: str | None = None,
         extra_index_urls: list[str] | None = None,
         skip_folder_validation: bool = False,
+        skip_validation: bool = False,
```

Comment:
Should we instead call it `skip_handle_validation` to avoid confusing it with the `skip_folder_validation` argument? What do you think?
```suggestion
        skip_handle_validation: bool = False,
```